dataprofiler/profilers/profile_builder.py

```diff
@@ -634,14 +634,15 @@ def tqdm(l):
         if options.structured_options.multiprocess.is_enabled:
             cpu_count = 1
             try:
+                mp.set_start_method('fork')
                 cpu_count = mp.cpu_count()
             except NotImplementedError as e:
                 cpu_count = 1
 
-            # No additional advantage beyond 8 processes
+            # No additional advantage beyond 5 processes
             # Always leave 1 cores free
             if cpu_count > 2:
-                cpu_count = min(cpu_count-1, 8)
+                cpu_count = min(cpu_count-1, 5)
```

Comment:
I think it is actually 4 right? The largest grouping is datetime, text, float, and int right?